### DISCUSSION TITLE
I reset the database and the players' skins are still saved.

### DISCUSSION THREAD

**Discussion by crashzk** [ianlucas/cs2-inventory-simulator#114] [General]
I reset the database, I didn't delete it, I really reset it, I deleted everything. However, even so, the players' inventories do not reset, they enter the website and the skins remain there.

Would this be a bug, or is it expected behavior? How can I reset all players' inventory? Make everyone start from scratch.

**Comment by ianlucas** [comment]
This is a feature, we cache the inventory for offline & not-authenticated users. You can open an issue requesting a rule for disabling inventory caching, but keep in mind it'll also prevent non-authenticated users from saving their inventory.

**Comment by crashzk** [comment]
Ok, but offline and unauthenticated users, if you enter the CS2 server, you don't have the skins, right?

To use the skins on the servers, using your plugin you need to log in, correct? If so, we can open a request to activate/deactivate this feature.

**Reply by ianlucas** [reply]
Yes, if you reset the database, and the user gets back to the app, they'll see their cached inventory, but it won't be available in Inventory Simulator's API as the users are not authenticated (thus their user doesn't exist in the database). Once they click to sync, the system will create an user and try to add the items from their cached inventories - only then the API will return their items (when equipped).

**Reply by crashzk** [reply]
In this case, I will open the request using the citation resource then. As I use it exclusively for CS2 servers, with the plugin it has, I don't see any point in using it offline and unauthenticated.

Anyway, opening the request, or would you prefer to convert this discussion to Issues?

**Reply by ianlucas** [reply]
Go ahead and open a request issue.
